tools/daca2.py

```diff
@@ -176,63 +177,49 @@ def scanarchive(filepath, jobs, cpulimit):
     results.write('\n')
     results.close()
 
-FOLDER = None
-JOBS = '-j1'
-REV = None
-SKIP = []
-WORKDIR = os.path.expanduser('~/daca2')
-CPULIMIT = None
-for arg in sys.argv[1:]:
-    if arg[:6] == '--rev=':
-        REV = arg[6:]
-    elif arg[:2] == '-j':
-        JOBS = arg
-    elif arg.startswith('--skip='):
-        SKIP.append(arg[7:])
-    elif arg.startswith('--workdir='):
-        WORKDIR = arg[10:]
-    elif arg.startswith('--cpulimit='):
-        CPULIMIT = arg[11:]
-    else:
-        FOLDER = arg
+parser = argparse.ArgumentParser(description='Checks debian source code')
+parser.add_argument('--folder', required=True)
```

Comment:
isn't it possible to keep the syntax for specifying folder? this will break my rundaca2.py scripts that I execute currently..